docs/source/basic_usage.rst

```diff
@@ -254,3 +254,20 @@ interact with it. Even better, you can **go back in time** because you know what
     vdb <your_json_report>
 
 For detailed commands, please refer to :doc:`virtual_debug`
+
+Compress Your Report
+--------------------
+
+VizTracer supports compressing your json report. The compress rate is better than commonly used compression methods like ``zlib`` and ``lzma``.
```

Comment:
The documentation does not need to say it's better than the common methods. However, we can give a general compress rate here.

Reply:
updated here!